src/Analyzer.JsonRuleEngine.UnitTests/InOperatorTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.Operators;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.UnitTests
+{
+    [TestClass]
+    public class InOperatorTests
+    {
+        [DataTestMethod]
+        [DataRow(true, new object[] { "anotherValue", "aValue" }, DisplayName = "String is in string array")]
+        [DataRow(false, new object[] { "anotherValue", "otherValue" }, DisplayName = "String is not in string array")]
+        [DataRow(false, new object[] { }, DisplayName = "String is not in empty array")]
+        [DataRow(true, new object[] { "anotherValue", 4, false, 3.5, null, "aValue" }, DisplayName = "String is in mixed array")]
+        [DataRow(false, new object[] { "anotherValue", 4, false, 3.5, null, "otherValue" }, DisplayName = "String is not in mixed array")]
+        public void EvaluateExpression(bool evaluationResult, params object[] arrayOfValues)
+        {
+            var aValue = "aValue";
```

Comment:
Add tests to find other data types inside the array as well (besides a string).

Reply:
I thought it wasn't necessary because for the comparison I'm using the 'equals' operator that covers those cases in its own tests, do you think I should add those tests anyways?

Reply:
Using the 'equals' operator is a good way to implement it.  Having tests ensures it's working as intended, and also protects against future changes that may inadvertently break it.